src/Microsoft.AspNet.OData/Extensions/HttpRequestMessageExtensions.cs

```diff
@@ -209,20 +209,26 @@ public static ETag<TEntity> GetETag<TEntity>(this HttpRequestMessage request, En
         /// <param name="pageSize">The number of results allowed per page.</param>
         /// <returns>A next page link.</returns>
         public static Uri GetNextPageLink(this HttpRequestMessage request, int pageSize)
```

Comment:
>public static Uri GetNextPageLink(this HttpRequestMessage request, int pageSize) [](start = 8, length = 80)

Do we expect to deprecate this in favor of the new overload? What happens if GetNextPageLink is called with null instance and objToSkipToken values? Does it default to the old behavior?  Same for the GetNextPageLink method on WebApiRequestMessage?